### ISSUE TITLE
Producing unique avatars based on some provided input

### ISSUE THREAD

**Issue by ashfame** [orgordin/vuejs-avataaars#7]
Hi, cool library!

Does this library expose the ability to generate the same avatar for the same input every single time? I couldn't do this on the demo.

**Comment by orgordin** [comment]
Yes it is. Same input will generate the same avatar. I will fix the demo :)

**Comment by ashfame** [comment]
Great! Can you summarize how it works? I am guessing svg components are shown or hidden based on the values of certain variables and those svg components are stacked upon each other in a particular order and that results into the final look. Would that be accurate?

**Comment by orgordin** [comment]
Yes, SVG depends on your props.
`    <avataaars
    :eyebrowType="'Angry'"
    :eyeType="'Cry'"
    :mouthType="'Eating'"
    :isCircle="false"
    :clotheColor="'Blue01'"
    :skinColor="'Light'"
    :facialHairColor="'Blonde'"
    ></avataaars>`
